projects.js

```diff
@@ -0,0 +1,164 @@
+        ${project.name}
+      </h3>
+
+      <div class="project-info-top">
+        <p class="job-place">
+          ${project.jobDescription.company}
+        </p>
+        <img class="circle" src="images/Counter.svg" alt="">
+        <p class="job-title gray">
+          ${project.jobDescription.role}
+        </p>
+        <img class="circle" src="images/Counter.svg" alt="">
+        <p class="project-year gray">
+          ${project.jobDescription.year}
+        </p>
+      </div>
+
+      <p class="project-text">
+        ${project.description}
+      </p>
+
+      <ul class="project-languages">
+        <li class="language">${project.languages[0]}</li>
+        <li class="language">${project.languages[1]}</li>
+        <li class="language">${project.languages[2]}</li>
+      </ul>
+
+      <button type="button" class="project-button">
+        See Project
+      </button>
+    </div>
+  </div>
+
+  `);
+  workSection.appendChild(projectHTML);
+});
+
+function createPopup(position) {
+  const popup = document.createElement('div');
+  popup.innerHTML = (`
+    <div class="popup-content">
+        <h3 class="project-title">
+           ${projects[position].name}
+        </h3>
+
+        <img class="icon-cancel" src="images/popup-images/Icon-Cancel-Gray.svg" alt="">
+        <img class="popup-project-img" src="images/SnapshootPortfolio(${position}).jpg" alt="">
+
+        <ul class="project-info-top">
+          <li class="job-place">${projects[position].jobDescription.company}</li>
+          <img class="circle" src="images/Counter.svg" alt="">
+          <li class="job-title gray">${projects[position].jobDescription.role}</li>
+          <img class="circle" src="images/Counter.svg" alt="">
+          <li class="project-year gray">${projects[position].jobDescription.year}</li>
+        </ul>
+
+        <img class="project-img-desktop" src="images/desktop-images/img${position + 1}.png" alt="project tonic homepage">
+
+        <div class="popup-bottom">
+          <p class="project-text">
+            ${projects[position].description}
+          </p>
+
+          <div class="badges-buttons">
+            <ul class="project-languages">
+              <li class="language">${projects[position].languages[0]}</li>
+              <li class="language">${projects[position].languages[1]}</li>
+              <li class="language">${projects[position].languages[2]}</li>
+            </ul>
+  
+            <div class="popup-buttons">
+              <button type="button" class="project-button">
+                See live
+                <img src="images/popup-images/Icon-Export.svg" alt="">
+              </button>
+              <button type="button" class="project-button">
+                See Source
+                <img src="images/popup-images/Icon -GitHub.svg" alt="">
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+  `);
+  popup.classList.add('popup');
+  document.body.appendChild(popup);
```

Comment:
- [ ] [Optional] I can see that you appended the popup modal directly in the HTML body so because of that you will see its div in the bottom of your body after the footer. Kindly It would be better semantically if you append it in the works section instead because this modal belongs to this section.

![image](https://user-images.githubusercontent.com/68796232/173326274-b5a0d6d9-d7bc-4108-8a4a-45253037159e.png)

### Other than that you have done a great job 💪.
**Kindly don't forgot the required change in the main message above**